docs/upgrade/v1-0-3-to-v1-1-1.md

```diff
@@ -0,0 +1,94 @@
+
+:::
+
+Once there is an upgradable version, the Harvester GUI Dashboard page will show an upgrade button. For more details, please refer to [start an upgrade](./automatic.md#start-an-upgrade).
+
+For the air-gap env upgrade, please refer to [prepare an air-gapped upgrade](./automatic.md#prepare-an-air-gapped-upgrade).
+
+
+## Known issues
+
+---
+
+### 1. An upgrade is stuck when pre-draining a node
+
+Starting from v1.1.0, Harvester will wait for all volumes to become healthy (when node counts >= 3) before upgrading a node. Generally, users can check volumes' health if an upgrade is stuck in the "pre-draining" state.
+
+Visit ["Access Embedded Longhorn"](../troubleshooting/harvester.md#access-embedded-longhorn) to see how to access the embedded Longhorn GUI.
+
+---
+
+### 2. An upgrade is stuck when pre-draining a node (case 2)
+
+An upgrade is stuck, as shown in the screenshot below:
+
+![](/img/v1.1/upgrade/known_issues/3216-stuck-pre-drain.png)
+
+And the user can also observe multiple nodes have scheduling disabled.
+
+```
+$ kubectl get nodes
+NAME    STATUS                     ROLES                       AGE   VERSION
+node1   Ready                      control-plane,etcd,master   20d   v1.24.7+rke2r1
+node2   Ready,SchedulingDisabled   control-plane,etcd,master   20d   v1.24.7+rke2r1
+node3   Ready,SchedulingDisabled   control-plane,etcd,master   20d   v1.24.7+rke2r1
+```
+
+- Related issue:
+  - [[BUG] Multiple nodes pre-drains in an upgrade](https://github.com/harvester/harvester/issues/3216)
+- Workaround:
+  - https://github.com/harvester/harvester/issues/3216#issuecomment-1328607004
+
+---
+
+### 3. Upgrade stuck in upgrading first node: Job was active longer than specified deadline
+
+An upgrade fail, as shown in the screenshot below:
+
+![](/img/v1.1/upgrade/known_issues/2894-deadline.png)
+
+
+- Related issue:
+  - [[BUG] Upgrade stuck in upgrading first node: Job was active longer than specified deadline](https://github.com/harvester/harvester/issues/2894)
+- Workaround:
+  - https://github.com/harvester/harvester/issues/2894#issuecomment-1274069690
+
+
+---
+
+### 4. An upgrade is stuck after a node is pre-drained
+
+An upgrade is stuck, as shown in the screenshot below:
+
+![](/img/v1.1/upgrade/known_issues/3021-stuck.png)
+
+
+- Related issue:
+  - [[BUG] Upgrade stop at upgrading node3 stage, stuck in Pre-drained status](https://github.com/harvester/harvester/issues/3021)
+- Workaround:
+  - https://github.com/harvester/harvester/issues/3021#issuecomment-1288747614
+
+---
+
+### 5. Additional certificates are gone after an upgrade
+
+After upgrading from v1.0.3, the file `/etc/pki/trust/anchors/additional-ca.pem` configured via the [`additioanl-ca` setting](https://docs.harvesterhci.io/v1.1/advanced/settings#additional-ca) is gone.
+
+To fix this, the user needs to access the Settings page (Harvester GUI, `Advanced -> Settings`): 
+- Edit the `additional-ca` setting. Back up the current value first and click `Save`.
```

Comment:
We may need to backup the current value, clean it up, and then `Save`, so we don't hit [the annotation check](https://github.com/harvester/harvester/blob/0cdb5a5b1cb104c825eddb88002e81648ff422dd/pkg/controller/master/setting/handler.go#L77-L79).

Reply:
Thanks, and add the "clear value" statement.